mobile/src/store/courses/actions.ts

```diff
@@ -0,0 +1,72 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { Course } from '~/common/types/courses/courses';
+import { AsyncThunkConfig } from '~/common/types/types';
+
+import { ActionType } from './common';
+
+const courses = [
```

Comment:
Move this to courses/data/mock-courses.ts
```suggestion
const MOCK_COURSES = [
```